runner/runners/bazel.go

```diff
@@ -79,20 +85,28 @@ func postProcessBazel(filer snapshot.Filer,
 
 	stdoutDigest, err := writeFileToCAS(bzFiler, stdout.AsFile())
 	if err != nil {
-		return nil, fmt.Errorf("Error ingesting stdout to CAS: %s", err)
+		errstr := fmt.Sprintf("Error ingesting stdout to CAS: %s", err)
+		log.Error(errstr)
+		return nil, fmt.Errorf("%s", errstr)
```

Comment:
nit: for all of these, can just fmt.Errorf(errstr)

Reply:
done, thanks